### PR TITLE
enhance: Make import-related error message clearer

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #28976

### CODE REVIEW

**Comment by xiaocai2333** [comment]
rerun ut

**Comment by xiaocai2333** [comment]
rerun ut

File: internal/util/importutil/binlog_adapter.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 	"go.uber.org/zap"
 
 	"github.com/milvus-io/milvus-proto/go-api/v2/schemapb"
+	"github.com/milvus-io/milvus/internal/querycoordv2/params"
```
Comment (czs007) [inline internal/util/importutil/binlog_adapter.go:29]: strange import

**Pull request merged**